src/BlockTypes/MiniCart.php

```diff
@@ -462,5 +462,16 @@ protected function register_block_type_assets() {
 		$chunks = preg_filter( '/$/', '-frontend', $blocks );
 
 		$this->register_chunk_translations( $chunks );
```

Comment:
Can we remove this line, right?

Reply:
My bad, it's my copy-pasta issue. Yes, we can, that's the purpose of this PR :D.